### ISSUE TITLE
String substitutions do not work

### ISSUE THREAD

**Issue by trajano** [onubo/react-native-logs#60]
[Using string substitutions](https://developer.mozilla.org/en-US/docs/Web/API/console#using_string_substitutions) being passed to the log does not work on consoleTransport

I presume it's because you're assembling the message and sending it out as one string.

**Comment by alessandro-bottamedi** [comment]
Yes, the only way to get the string subtitution to work is to create a custom transport and process the rawMsg...  perhaps you could create a customTransport that directly calls console.log without processing the message.
